ReactWindows/Playground/index.windows.js

```diff
@@ -11,6 +11,12 @@ import {
   View,
 } from 'react-native';
 
+require('react-native/Libraries/BatchedBridge/MessageQueue').spy(info => {
+  console.log(`${!info.type ? 'N->JS' : 'JS->N'} : `
```

Comment:
did you mean to leave this in here?

Reply:
That was for testing. I'll revert changes in this file.